packages/components/src/components/swipeable-notification/swipeable-notification.scss

```diff
@@ -0,0 +1,55 @@
+.swipeable-notification {
+    width: 100%;
+    min-height: 4.8rem;
+    border-radius: $BORDER_RADIUS;
+    padding: 0.8rem;
+    background-color: $color-grey-5;
+    display: flex;
+    justify-content: space-between;
+    text-decoration: none;
+
+    * {
+        color: $color-black-1; // text color is unchanged for both themes
```

Comment:
Not using the color prop in those 3 Text components I'm using since <Text doesn't support the same color for both themes. <Text component was meant to adapt to different themes, so it doesn't feel right to me to add an extra constant which would maintain the same color for both themes. Please share your opinion if you disagree :)